### ISSUE TITLE
TypeError comparing complex type and float type in `is_close`-like functions

### ISSUE THREAD

**Issue by liwt31** [cupy/cupy#2230]
* Conditions
```
CuPy Version          : 5.4.0
CUDA Root             : /usr/local/cuda-10.1
CUDA Build Version    : 10010
CUDA Driver Version   : 10010
CUDA Runtime Version  : 10010
cuDNN Build Version   : 7500
cuDNN Version         : 7500
NCCL Build Version    : 2402
NCCL Runtime Version  : 2402
```
* Code to reproduce
```
import cupy as cp
a = cp.random.rand(10)
b = a * 1j
cp.isclose(a, b)
```
* Error messages, stack traces, or logs
```
Traceback (most recent call last):
  File "test.py", line 4, in <module>
    cp.isclose(a, b)
  File "/home/wtli/.conda/envs/cp/lib/python3.6/site-packages/cupy/logic/comparison.py", line 73, in isclose
    return _is_close(a, b, rtol, atol, equal_nan)
  File "cupy/core/_kernel.pyx", line 807, in cupy.core._kernel.ufunc.__call__
  File "cupy/core/_kernel.pyx", line 692, in cupy.core._kernel._guess_routine
TypeError: Wrong type ((<class 'numpy.float64'>, <class 'numpy.complex128'>, <class 'numpy.float64'>, <class 'numpy.float64'>, <class 'numpy.bool_'>)) of arguments for cupy_is_close
```
Apparently similar functions like `allclose` have the same behavior.

I know it's logically not sound to compare complex value and float value directly, but **in `numpy` no error will be raised in such cases.** 

Thanks!

**Comment by leofang** [comment]
Duplicate of #1946. Please update your CuPy version to 6.0.0. This has been resolved in PR #1947.

**Comment by liwt31** [comment]
Updating solves the issue. Thanks!
